src/app/ai/mcp-server/tools/c2Registry/chat-creation.ts

```diff
@@ -0,0 +1,62 @@
+import { IMCPTool, IRegistryServiceTools } from '../../../mcp-models';
+
+const symbol = 'IChatCreationService';
+
+const createRoomParams = {
+  type: 'object',
+  properties: {
+    users: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          id: {
+            type: 'integer',
+          },
+        },
+        required: ['id'],
+      },
+      description:
+        'The users to create the chat room with (default value: empty array)',
+    },
+    roomName: {
+      type: 'string',
+      description:
+        'The name of the chat room to create (default value: empty string)',
+    },
+    roomSettings: {
+      type: 'object',
+      description: `The settings of the chat room to create (default value: empty object). Here is the list of available settings:
```

Comment:
Wow room settings without properties and it works just thanks to the description? Incredible